xarray/tests/test_backends.py

```diff
@@ -5577,3 +5577,8 @@ def test_raise_writing_to_nczarr(self, mode):
                 KeyError, match="missing the attribute `_ARRAY_DIMENSIONS`,"
             ):
                 ds.to_zarr(tmp, mode=mode)
+
+
+def test_pickle_open_mfdataset_dataset():
```

Comment:
```suggestion
@requires_netCDF4
def test_pickle_open_mfdataset_dataset():
```

Reply:
Should fix the tests

Reply:
Thank you! I have never written a test before, and was unsure if I needed any decorators. The test ran and passed locally when I did `pytest xarray/test/test_backends.py -k test_pickle_open_mfdataset_dataset` but that was just for my environment. I'll add this in and trigger CI again and see what happens.